### PR TITLE
Capabilities management

### PR DESCRIPTION
**Pull request by erezzarum**
*Issue #, if available:*

*Description of changes:*
Add support to enable/disable capabilities from the global tfvars

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: single-account-single-cluster-multi-env/30.eks/30.cluster/main.tf [hunk 1]
```diff
@@ -27,9 +27,12 @@ module "eks" {
   cluster_version                = local.cluster_version
   cluster_endpoint_public_access = try(!var.cluster_config.private_eks_cluster, false)
 
+  cluster_enabled_log_types      = ["audit", "api", "authenticator", "controllerManager", "scheduler"]
+
   vpc_id                   = data.terraform_remote_state.vpc.outputs.vpc_id
   subnet_ids               = data.terraform_remote_state.vpc.outputs.private_subnet_ids
-  control_plane_subnet_ids = data.terraform_remote_state.vpc.outputs.intra_subnet_ids
+  control_plane_subnet_ids = data.terraform_remote_state.vpc.outputs.private_subnet_ids
+  # control_plane_subnet_ids = data.terraform_remote_state.vpc.outputs.intra_subnet_ids
```
Comment (erezzarum) [inline single-account-single-cluster-multi-env/30.eks/30.cluster/main.tf:35]: Ignore this for now, just for testing.

File: single-account-single-cluster-multi-env/30.eks/30.cluster/main.tf [hunk 2]
```diff
@@ -39,7 +44,62 @@ module "eks" {
   ))
 
   enable_cluster_creator_admin_permissions = "true"
-  authentication_mode                      = "API_AND_CONFIG_MAP"
+  authentication_mode                      = try(var.cluster_config.authentication_mode, "API_AND_CONFIG_MAP")
+
+  bootstrap_self_managed_addons = "false"
+
+  cluster_addons = merge(
```
Comment (tsahiduek) [inline single-account-single-cluster-multi-env/30.eks/30.cluster/main.tf:51]: Why we've added the addon into the EKS module and not the EKS blueprints addons module?

File: single-account-single-cluster-multi-env/30.eks/30.cluster/main.tf [hunk 3]
```diff
@@ -219,48 +280,9 @@ module "eks_blueprints_addons" {
   # We want to wait for the Fargate profiles to be deployed first
   create_delay_dependencies = [for prof in module.eks.fargate_profiles : prof.fargate_profile_arn]
 
-  eks_addons = {
```
Comment (tsahiduek) [inline single-account-single-cluster-multi-env/30.eks/30.cluster/main.tf:222]: can we keep the addons in the EKS Blueprints module and not the EKS one?

**Review by tsahiduek — APPROVED** [review]
lgtm

**Pull request merged**